test/test_conversion.py

```diff
@@ -10,8 +10,8 @@
 import sbol3
 
 from sbol_utilities.conversion import convert2to3, convert3to2, convert_to_genbank, convert_to_fasta, \
-    convert_from_fasta, convert_from_genbank, \
-    main, sbol2fasta, sbol2genbank, sbol2to3, sbol3to2, fasta2sbol, genbank2sbol
+    convert_from_fasta, convert_from_genbank, main \
```

Comment:
Does this change need to be there? It is better to not change code if your change does not improve it, as that just complicates understanding its history.

Reply:
Yeah, you are right.
I updated it